crates/aptos/src/common/types.rs

```diff
@@ -1255,7 +1255,7 @@ impl TransactionOptions {
                     simulated_txn.info.vm_status
                 )));
             }
-            simulated_txn.info.gas_used.0
+            simulated_txn.info.gas_used.0 + 1
```

Comment:
@gregnazario it seems like there's some sort of off-by-1 error in the gas estimation process. Without the "+1" here, `test_join_and_leave_validator` would fail due to running out of gas.